### PR TITLE
Fix description

### PR DESCRIPTION
**Pull request by sawa-zen**
@rrreeeyyy 

Fixed that HTML tags are not included in headline and `og:description`.

### CODE REVIEW

File: app/controllers/index_controller.rb [hunk 1]
```diff
@@ -24,8 +24,8 @@ class IndexController < BaseController
     @entry = Entry.find_by_id!(id)
     halt(404) unless @entry.published
 
-    @description = @entry.content.truncate(100)
     @entry.content = markdown.render(@entry.content) if @entry.content
+    @description = Nokogiri::HTML(@entry.content).inner_text.truncate(100)
```
Comment (rrreeeyyy) [inline app/controllers/index_controller.rb]: Why do you create `@description` here? Where is this variable use?

File: app/views/module/_entries.erb [hunk 2]
```diff
@@ -14,7 +14,7 @@
       </div>
       <!-- ヘッドライン -->
       <div class="headline">
-        <%= entry.content.truncate(100) %>
+        <%= Nokogiri::HTML(entry.content).inner_text.truncate(100) %>
```
Comment (rrreeeyyy) [inline app/views/module/_entries.erb]: IMHO: It should be avoided to write a logic in erb template.
If I were you, I would implement `summarize` method to `Entry` model.
Then call `entry.summarize` method in the template.

File: app/controllers/index_controller.rb (hunk 1 above)
Comment (sawa-zen) [inline app/controllers/index_controller.rb]: I use it in this code.
https://github.com/topotal/blog/blob/d8755c2ebc21c8e6e5dd879c6c2ecd7346b3db17/app/views/layout.erb#L24

**Comment by sawa-zen** [comment]
@rrreeeyyy 
Please review it again.

**Comment by rrreeeyyy** [comment]
LGTM. good works.

**Pull request merged**